src/runners/cairoRunner.ts

```diff
@@ -43,23 +43,21 @@ export class CairoRunner {
 
   // Run until the given PC is reached.
   runUntilPc(finalPc: Relocatable, verbose: boolean = false): void {
-    while (this.vm.runContext.pc.getOffset() < finalPc.getOffset()) {
+    while (this.vm.pc.getOffset() < finalPc.getOffset()) {
       this.vm.step();
       if (verbose) {
         console.log(
-          `AP: ${this.vm.runContext.ap.getSegmentIndex()}:${this.vm.runContext.ap.getOffset()}`
+          `AP: ${this.vm.ap.getSegmentIndex()}:${this.vm.ap.getOffset()}`
```

Comment:
we could have a `toString()` for relocatable

Reply:
true

Reply:
adding to the new issue